ios/Video/Features/RCTResourceLoaderDelegate.swift

```diff
@@ -135,7 +158,7 @@ class RCTResourceLoaderDelegate: NSObject, AVAssetResourceLoaderDelegate, URLSes
             ) .then{ spcData -> Void in
                 self._requestingCertificate = true
                 self._onGetLicense?(["licenseUrl": self._drm?.licenseServer ?? "",
-                                     "contentId": contentId,
```

Comment:
As this is a breaking change, it would be better to keep contentId as is and add a new property for license url 
(loadingRequest.request.url?.absoluteString).
It will make the update easier for client applications

Reply:
We can change the licenseUrl parameter as you defined it in  https://github.com/react-native-video/react-native-video/issues/2881#issuecomment-1416190805 and keep contentId as is.